src/webots/nodes/WbPositionSensor.cpp

```diff
@@ -52,6 +52,7 @@ WbPositionSensor::WbPositionSensor(const WbNode &other) : WbJointDevice(other) {
 void WbPositionSensor::init() {
   mSensor = new WbSensor();
   mValue = 0.0;
+  mValueVel = 0.0;
```

Comment:
We try to avoid abbreviations:
```suggestion
  mVelocityValue = 0.0;
```